mlrun/api/crud/model_monitoring/helpers.py

```diff
@@ -23,6 +24,16 @@
 import mlrun.common.schemas.schedule
 import mlrun.errors
 
+Seconds = typing.NewType("Seconds", int)
+Minutes = typing.NewType("Minutes", int)
+
+_SECONDS_IN_MINUTE: Seconds = 60
+_MINUTES_IN_HOUR: Minutes = 60
+
+
+def seconds2minutes(seconds: Seconds) -> Minutes:
```

Comment:
At the moment this function gets integer values and not `Seconds`. In other words, you need to wrap the values on the caller side to `Seconds` or change the docstrings here to `int`.

Reply:
Done - 2959854